bin/check_perms

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+set -e
+set -u
+export PATH=/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin
```

Comment:
Is this actually executed on local? This somehow overrides my path envs in the current terminal session.

Reply:
@ragnarkurmwunder ^

Reply:
I don't really know how this works. Its something that probably could be separately fixed.

Reply:
At least It doesn't seem to be overriding, I tested in my local